### ISSUE TITLE
Error on onPostBuild lifecycle: Package subpath './dist/utils' is not defined

### ISSUE THREAD

**Issue by gablittledevshop** [percy/gatsby-plugin-percy#102]
## The problem

Percy fails when I run the gatsby build command. It seems that `@percy/cli-snapshot/dist/utils` is missing.

## Environment

- Node version: v16.14.2
- `@percy/cli` version: 1.1.0
- Version of Percy SDK you’re using: 1.0.0
- If needed, a build or snapshot ID: N/A
- OS version: Debian 3.10-bullseye (Docker container https://github.com/microsoft/vscode-dev-containers/tree/v0.217.4/containers/python-3) 
- Type of shell command-line [interface]: Bash shell

## Details
Gatsby version: 4.3.0

I added `gatsby-plugin-percy` to my project following the instructions here https://docs.percy.io/docs/gatsby. For the gatsby-config settings, I just have `gatsby-plugin-percy` set without the additional options.

Running my build command I get this error:

```
"gatsby-plugin-percy" threw an error while running the onPostBuild lifecycle:

Package subpath './dist/utils' is not defined

  51 |
  52 |   // lazily required to not bloat the process when not building
> 53 |   let { serve, mapStaticSnapshots } = require('@percy/cli-snapshot/dist/utils');
     |                                       ^
  54 |   let utils = require('@percy/sdk-utils');
  55 |
  56 |   if (!(await utils.isPercyEnabled())) {

Make sure that you don't have a typo somewhere and use valid arguments in onPostBuild lifecycle.
Learn more about onPostBuild here: https://www.gatsbyjs.com/docs/reference/config-files/gatsby-node/#onPostBuild

File: node_modules/gatsby-plugin-percy/gatsby-node.js:53:39
```

**Comment by gablittledevshop** [comment]
An update on this, I tried to install `@percy/cli` at version `1.0.0-beta.75` which fixes the above issue, but then there's an issue with the `require`:

```
require() of ES Module /workspaces/project/node_modules/@percy/sdk-utils/dist/index.js from
/workspaces/project/node_modules/gatsby-plugin-percy/gatsby-node.js not supported.
Instead change the require of index.js in /workspaces/bbtv.com/node_modules/gatsby-plugin-percy/gatsby-node.js
to a dynamic import() which is available in all CommonJS modules.

  52 |   // lazily required to not bloat the process when not building
  53 |   let { serve, mapStaticSnapshots } = require('@percy/cli-snapshot/dist/utils');
> 54 |   let utils = require('@percy/sdk-utils');
     |               ^
  55 |
  56 |   if (!(await utils.isPercyEnabled())) {
  57 |     return activity.end('Disabled, skipping snapshots');

File: node_modules/gatsby-plugin-percy/gatsby-node.js:54:15

  Error: [ERR_REQUIRE_ESM]: require() of ES Module /workspaces/bbtv.com/node_modules/@percy/sdk-utils/dist/index
  .js from /workspaces/bbtv.com/node_modules/gatsby-plugin-percy/gatsby-node.js not supported.
  Instead change the require of index.js in /workspaces/bbtv.com/node_modules/gatsby-plugin-percy/gatsby-node.js
   to a dynamic import() which is available in all CommonJS modules.
```

**Comment by JKlause** [comment]
We're having the same issue. Any update on this?

**Comment by Jsan2020** [comment]
I get the same error using the updated `@percy/cli@1.2.1`

**Comment by johndrwood** [comment]
Same error on `@percy/cli@1.3.0`

**Comment by johndrwood** [comment]
> You should be able to downgrade to any of the betas for CLI and have it work

I get the same error for `1.0.0-beta.76` and `1.0.0-beta.75`. Is there another beta version you think would work?

**Comment by tim-line** [comment]
Is there any update on this or can someone confirm a workaround for a Gatsby site deployed to Netlify?

**Comment by wwilsman** [comment]
Hi @tim-line! We're actively working on the next version of this SDK, so watch for a new release in the next week or so!

In the meantime, the `percy snapshot` command can take snapshots of virtually any site. You can point it at your static build output, a hosted sitemap, or a file containing a list of URLs to snapshot. https://docs.percy.io/docs/cli-snapshot

**Comment by Robdel12** [comment]
Fixed with v1.1.0 🥳
